src/main/java/org/spongepowered/common/mixin/core/item/MixinEnchantment.java

```diff
@@ -44,8 +44,8 @@
 
 @NonnullByDefault
 @Mixin(net.minecraft.enchantment.Enchantment.class)
-@Implements(@Interface(iface = Enchantment.class, prefix = "enchantment$"))
-public abstract class MixinEnchantment implements Enchantment, IMixinEnchantment {
+@Implements(@Interface(iface = EnchantmentType.class, prefix = "enchantment$"))
+public abstract class MixinEnchantment implements IMixinEnchantment {
```

Comment:
Why remove it? It'll still perfectly implement just fine normally.

Reply:
`IMixinEnchantment` extends `Enchantment` so I didn't think it was needed.

Reply:
Thinking about it further, the `IMixin` interfaces shouldn't extend the API interfaces for the basic understanding that they are only from implementation, The only reason why there's `IMixinEntity` and a few others doing this is because of Java compiler requirements with potentially ambiguous methods.